lib/cylc/cylc_xdot.py

```diff
@@ -403,7 +400,7 @@ def get_graph(self, group_nodes=[], ungroup_nodes=[],
         graph.graph_attr['rankdir'] = self.orientation
 
         for node in graph.nodes():
-            name, point_string = TaskID.split(node.get_name())
+            name, _ = TaskID.split(node.get_name())
```

Comment:
(I'll do this on the basis that it just looks nicer to not have ignored/dummy variables).